test/project_types/extension/forms/questions/ask_template_test.rb

```diff
@@ -45,11 +43,29 @@ def test_ctx_is_the_only_required_configuration_option
           assert_nothing_raised { AskTemplate.new(ctx: FakeContext.new) }
         end
 
+        def test_user_provided_template_is_chosen_over_interactive_option
+          mock_beta(true)
```

Comment:
I think readability could be a improved using keyword arguments here:

```ruby
mock_environment_settings(shopifolk: true, extensions_server_beta: true)
```

Its wordier but results in not having to navigate to `mock_beta` in order to understand what's going on. Alternatively, just implement a method called `opt_into_extension_server_beta` and have it take no arguments.